topi/python/topi/cuda/nms.py

```diff
@@ -151,8 +151,7 @@ def calculate_overlap(out_tensor, box_a_idx, box_b_idx):
             with ib.if_scope(tvm.all(nms_topk_node > 0, nms_topk < p_valid_count[b])):
                 with ib.for_range(0, p_valid_count[b] - nkeep) as l:
                     with ib.if_scope(i < 6):
-                        p_out[(base_idx + (l + nkeep) * 6 + i)] = \
-                                p_data[(base_idx + (l + nkeep) * 6 + i)]
+                        p_out[(base_idx + (l + nkeep) * 6 + i)] = -1.0
```

Comment:
What's the meaning of -1? Are the last (valid_count - nkeep) bboxes dropped by this line?

Reply:
Yes, it's based on the implementation of the GluonCV SSD, plz reference PR #2353 .